card/mod/history/set/all/act_view.rb

```diff
@@ -0,0 +1,49 @@
+format :html do
+  def default_act_args args
+    act = (args[:act]  ||= Act.find(params["act_id"]))
+    args[:act_seq]     ||= params["act_seq"]
+    args[:hide_diff]   ||= hide_diff?
+    args[:slot_class]  ||= "revision-#{act.id} history-slot list-group-item"
+    args[:action_view] ||= action_view
+    args[:actions]     ||= action_list args
+    act_context args
+  end
+
+
+  view :act do |args|
+    ActRenderer.new(self, args[:act], args).render
```

Comment:
this is used to render recent changes and history. `args[:act_context]` switches between those two
`absolute` = recent changes
`relative` = history